UI/DevScreens.cpp

```diff
@@ -878,12 +880,14 @@ void DrawProfile(UIContext &ui) {
 		if (area)
 			col = opacity | (col & 0xFFFFFF);
 		UI::Drawable color(col);
+		UI::Drawable outline(opacity | 0xFFFFFF);
 
 		if (area) {
 			for (int n = 0; n < historyLength; n++) {
 				float val = history[n];
 				float valY1 = ui.GetBounds().y2() - 10 - (val + total[n]) * scale;
 				float valY2 = ui.GetBounds().y2() - 10 - total[n] * scale;
+				ui.FillRect(outline, Bounds(x, valY1, dx, valY2 - valY1 + 0.5f));
```

Comment:
Hm, this will cover nearly exactly the same pixels as the subsequent call, just an extra line or so (so it does produce the desired white line), so it will burn lots of fillrate (important on mobile) and reduces the alpha transparency, which I wanted to keep.